frontend/src/stimulus/controllers/hover-card-trigger.controller.ts

```diff
@@ -60,9 +60,25 @@ export default class HoverCardTriggerController extends ApplicationController {
   private cardMouseLeaveBound= this.onMouseLeave.bind(this);
   private cardMouseEnterBound= this.onMouseEnter.bind(this);
 
+  /**
+   * Attempting to show hover cards on touch-based devices often leads to quirky and unwanted
+   * behavior. As all information in our hover cards is optional, we'd rather only show them
+   * on devices that primarily support a mouse cursor. That means that hovering is a reliable
+   * option for interaction.
+   */
+  shouldRegisterHoverEventListeners() {
```

Comment:
It would be more efficient to check this once and cache the boolean result of this method in a field. If set, we can skip this check, since neither browser, nor environment will change once the page has been loaded.

But in the end this is one more hoop to jump through and I could not bring myself to do it. What do you think? Is it worth it?

Reply:
The function is so simple, I don' think that it is worth it.